### PR TITLE
Replace Harbor with GSP ImageRepositories

### PR DESCRIPTION
**Pull request by Krenair**

### CODE REVIEW

File: ci/sandbox/release-pipeline.yaml [hunk 1]
```diff
@@ -239,7 +244,6 @@ spec:
               tar -rf alpine-image/rootfs.tar vmc-chart-package/
 
       - put: release
-        get_params: {skip_download: true}
         params:
-          import_file: alpine-image/rootfs.tar
-          <<: *image_tag
+          image: alpine-image/rootfs.tar
+          additional_tags: src/.git/short_ref
```
Comment (samcrang) [inline ci/sandbox/release-pipeline.yaml]: You could add a newline here if you can be bothered.

File: ci/sandbox/release-pipeline.yaml [hunk 2]
```diff
@@ -124,10 +129,8 @@ spec:
           - get: vmc-image
             passed: [build]
             trigger: true
-            params: {skip_download: true}
           - get: cloudhsm-image
```
Comment (chrisfarms) [inline ci/sandbox/release-pipeline.yaml]: does this image ever get built? :thinking: 

I think there is a missing build-step here ... I suspect the old pipeline was pulling an image build from another project's pipeline (proxy-node?).

I think we need to add a build step for it here really, it's a bit weird to be stealing the image from another project (even if they are somewhat intertwined)

Comment (Krenair) [reply ci/sandbox/release-pipeline.yaml]: It seems you're right - the verify-proxy-node pipeline put to a cloudhsm-client-image resource that had a repository that matches the one here. The two tenants have separate image repositories now so we will need to build it separately.

File: ci/sandbox/release-pipeline.yaml [hunk 3]
```diff
@@ -106,11 +108,43 @@ spec:
...
+            source:
+              repository: vito/oci-build-task
+            version:
+              digest: sha256:cfb2983956145f54a4996c2aff5fc598856c8722922a6e73f9ebfa3d9b3f9813
+          params:
+            CONTEXT: src
+          inputs:
+          - name: src
+          outputs:
+          - name: image
```
Comment (blairboy362) [inline ci/sandbox/release-pipeline.yaml]: While it's in our mind do we want to use per-image volumes to avoid the future gotcha?

**Comment by Krenair** [comment]
I think we're going to need to sort out alpine-image the same way we did in verify-proxy-node

File: ci/sandbox/release-pipeline.yaml [hunk 4]
```diff
@@ -106,11 +115,37 @@ spec:
...
+            path: image
       - put: vmc-image
-        get_params: {skip_download: true}
         params:
-          build: src
-          <<: *image_tag
+          image: vmc-image-file/image.tar
+          additional_tags: src/.git/short_ref
+
+      - build-cloudhsm-client
```
Comment (Krenair) [inline ci/sandbox/release-pipeline.yaml]:
```suggestion
      - task: build-cloudhsm-client
```

File: ci/sandbox/release-pipeline.yaml [hunk 5]
```diff
@@ -106,11 +115,37 @@ spec:
...
+          - name: vmc-image-file
+            path: image
       - put: vmc-image
-        get_params: {skip_download: true}
         params:
-          build: src
-          <<: *image_tag
+          image: vmc-image-file/image.tar
+          additional_tags: src/.git/short_ref
+
```
Comment (Krenair) [inline ci/sandbox/release-pipeline.yaml]:
```suggestion
      - get: cloudhsm-config
        trigger: true

```

File: ci/sandbox/release-pipeline.yaml [hunk 6]
```diff
@@ -106,11 +115,37 @@ spec:
...
+        config:
+          <<: *build_config
+          params:
+            CONTEXT: cloudhsm-config/cloudhsm
+          inputs:
+          - name: cloudhsm-config
+          outputs:
+          - name: cloudhsm-client-image-file
+            path: image
+      - put: cloudhsm-client-image
```
Comment (Krenair) [inline ci/sandbox/release-pipeline.yaml]:
```suggestion
      - put: cloudhsm-image
```

**Pull request merged**